src/components/Form.js

```diff
@@ -241,6 +241,15 @@ function Form(_props) {
       };
     });
   }, []);
+  const resetForm = ({ values, errors } = {}) => {
+    setState({
+      values: values ?? initialValues,
+      errors: errors ?? initialErrors ?? {},
+      shouldValidateOnChange: false,
+      namesToValidate: null,
+      submitStatus: "READY",
```

Comment:
I'm still confused why we set `shouldValidateOnChange`, `namesToValidate` and `submitStatus` here.
What happens if we don't set them (i.e. set only `values` and `errors`)?

Reply:
nothing really.   namesToValidate array hangs around but has no effect

it was more the idea that resetForm, sets the form back to it's initial state, so didn't see the issue in being explicit.

Reply:
> namesToValidate array hangs around but has no effect

I'm confused. Do you mean that errors are not getting updated? Might be easier to chat about this.

Also, by allowing passing an argument to `resetForm`, we create an overlap with `setValues` and `setErrors`.
Do you have a use case where you need this argument, but using `setValues` and `setErrors` is not enough?